.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: 
+  node_js
+node_js:
+  - "10.13.0"
+services:
+  - sequelize
```

Comment:
Is this line really necessary?

Reply:
Which exactly?

Reply:
i thought i should add it there since we are going to be using the ORM. Correct me if i am wrong

Reply:
[Check this out](https://docs.travis-ci.com/user/database-setup/#postgresql)